manuscript.Rmd

```diff
@@ -507,11 +525,38 @@ The point is that we cannot exclude it with certainty.
 This uncertainty is drastically reduced if the researchers have described what they intended to do beforehand and then report that they did exactly that.
 In that case, readers can be certain they received a complete account of the situation.
 They still might be uncertain about the actual theoretical risk the authors took, but to a much smaller extent than if the study would not have been preregistered.
+
 The remaining sources of uncertainty might be unfamiliarity with statistical methods or experimental paradigms used, the probability of an implementation error in the statistical analyses, a bug in the software used for analyses, etc.
+To further reduce the uncertainty about theoretical risk, researchers must therefore publish code and ideally data.
+After all, computational reproducibility is only possible if the data analytic procedure was communicated clearly enough to allow others to retrace the computational steps [@peikertReproducibleDataAnalysis2021].
+
 In any case, a well-written preregistration should aim to reduce the uncertainty about the theoretical risk and hence increase the persuasiveness of evidence.
 Therefore, a study that perfectly adhered to its preregistration will resemble the solid line in Figure \ref{fig:posterior}/\ref{fig:other-measures}.
 Crucially, perfect means here that the theoretical risk can be judged with low uncertainty, not that the theoretical risk is necessarily high.
 
+# Hacking, harking, and other harms
```

Comment:
I really like those examples!!!
One thing where I am not completely shure is Andreas comment - I would agree with the intuition that the way it is currently phrased, the first example does not have zero, but low theoretical risk.